packages/react-scripts/utils/isReactScriptsLinked.js

```diff
@@ -0,0 +1,7 @@
+var path = require('path');
+var fs = require('fs');
+
+module.exports = () => {
+  var localReactScriptsPath = path.resolve('node_modules/react-scripts');
+  return fs.existsSync(localReactScriptsPath) && fs.lstatSync(localReactScriptsPath).isSymbolicLink();
```

Comment:
`exists` is deprecated, but `existsSync` [is not](https://nodejs.org/dist/latest-v6.x/docs/api/fs.html#fs_fs_existssync_path).
See also https://github.com/facebookincubator/create-react-app/pull/1289.